src/wrapped_radix_engine_toolkit/high_level/manifest_building/third_party_deposit_update/manifest_third_party_deposit_update.rs

```diff
@@ -0,0 +1,150 @@
+use crate::prelude::*;
+
+use radix_engine_interface::blueprints::account::{
+    ACCOUNT_ADD_AUTHORIZED_DEPOSITOR, ACCOUNT_REMOVE_AUTHORIZED_DEPOSITOR,
+    ACCOUNT_REMOVE_RESOURCE_PREFERENCE_IDENT,
+    ACCOUNT_SET_DEFAULT_DEPOSIT_RULE_IDENT,
+    ACCOUNT_SET_RESOURCE_PREFERENCE_IDENT,
+};
+use transaction::prelude::ManifestBuilder as ScryptoManifestBuilder;
+
+impl TransactionManifest {
+    pub fn third_party_deposit_update(
+        owner: &AccountAddress,
+        from: ThirdPartyDeposits,
+        to: ThirdPartyDeposits,
+    ) -> Self {
+        let delta = ThirdPartyDepositsDelta::new(from, to);
+        Self::third_party_deposit_update_by_delta(owner, delta)
+    }
+
+    pub fn third_party_deposit_update_by_delta(
+        owner: &AccountAddress,
+        delta: ThirdPartyDepositsDelta,
+    ) -> Self {
+        let mut builder = ScryptoManifestBuilder::new();
```

Comment:
Manifest building is, as you can see, pretty nice and straightforward now that I've impl "automatic conversion" of Sargon types -> Scrypto types.